docs/excel/excel-add-ins-workbooks.md

```diff
@@ -257,12 +257,60 @@ The Excel API also lets add-ins turn off calculations until `RequestContext.sync
 context.application.suspendApiCalculationUntilNextSync();
 ```
 
+## Comments
+
+> [!NOTE]
+> The comment APIs are currently available only in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
+
+All the [comments](https://support.office.com/article/insert-comments-and-notes-in-excel-bdcc9f5d-38e2-45b4-9a92-0b2b5c7bf6f8) added to a workbook are tracked by the `Workbook.comments` property. This is a [CommentCollection](/javascript/api/excel/excel.commentcollection) object, which maintains a collection of [Comment](/javascript/api/excel/excel.comment) objects. This collection contains comments created by the user and comments created by your add-in.
+
+Comments are added to the workbook with the `CommentCollection.add` method. This method takes in the comment's text, as a string, and the cell where the comment will go, as a string or [Range](/javascript/api/excel/excel.range) object. The following code sample adds a comment to cell **A2**.
```

Comment:
Consider changing "where the comment will go" -> "where the comment should be added"

Reply:
Actually, you might consider combining the first two sentences, something like ->

You can add comments to a workbook by using the `CommentCollection.add` method, passing in the comment's text, as string, and the cell where the comment should be added, as either a string or [Range](/javascript/api/excel/excel.range) object.